googleapis/google-cloud-node
# migrate code from googleapis/nodejs-domains

- chore: add initial files for launch
- feat!: initial stub of library
- chore: add tests and samples to library (#2)
- chore: release 1.0.0 (#1)
- chore: update package name in config (#8)
- build: update gapic-generator-typescript to v1.2.10. (#10)
- chore: migrate to owl bot (#11)
- chore(deps): update dependency sinon to v10 (#17)
- chore(deps): update dependency @types/sinon to v10 (#26)
- chore(deps): update dependency ts-loader to v9 (#30)
- fix(deps): require google-gax v2.12.0 (#34)
- chore: new owl bot post processor docker image (#36)
- fix: use require() to load JSON protos (#37)
- chore: update gapic-generator-typescript to v1.3.2 (#38)
- chore: release 1.0.1 (#35)
- chore(deps): update dependency sinon to v11 (#41)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#42)
- chore: release 1.0.2 (#43)
- fix: make request optional in all cases (#49)
- chore: release 1.0.3 (#50)
- fix(deps): google-gax v2.17.0 with mTLS (#53)
- chore: release 1.0.4 (#54)
- fix(deps): google-gax v2.17.1 (#56)
- chore: release 1.0.5 (#57)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#58)
- chore: release 1.0.6 (#59)
- chore(nodejs): update client ref docs link in metadata (#64)
- fix(build): migrate to using main branch (#66)
- chore: release 1.0.7 (#67)
- fix(deps): google-gax v2.24.1 (#68)
- chore: release 1.0.8 (#69)
- feat: turns on self-signed JWT feature flag (#70)
- chore: release 1.1.0 (#71)
- build(node): update deps used during postprocessing (#1243) (#80)
- feat: added library for Cloud Domains v1 API. Also added methods for the transfer-in flow
- chore: release 1.2.0 (#82)
- chore: fix the wrong post processor image (#84)
- chore(deps): update dependency @types/node to v16 (#83)
- chore(cloud-rad): delete api-extractor config (#85)
- chore(deps): update dependency sinon to v12 (#86)
- docs(samples): add example tags to generated samples (#87)
- build: add generated samples to .eslintignore (#88)
- docs(node): support "stable"/"preview" release level (#1312) (#91)
- chore: add api_shortname and library_type to repo metadata (#90)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#93)
- test(nodejs): remove 15 add 16 (#1322) (#95)
- build: update copyright year to 2022 (#98)
- chore: update v2.12.0 gapic-generator-typescript (#100)
- chore(deps): update dependency sinon to v13 (#102)
- docs(nodejs): version support policy edits (#1346) (#104)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#105)
- chore: update v2.14.2 gapic-generator-typescript (#109)
- build(node): update client library version in samples metadata (#1356) (#123)
- chore(deps): update dependency @types/mocha to v9 (#124)
- chore(deps): update dependency sinon to v14 (#127)
- feat!: make v1 default service version (#110)
- build!: update library to use Node 12 (#129)
- chore(main): release 2.0.0 (#128)
- chore(deps): update dependency jsdoc-fresh to v2 (#133)
- chore(deps): update dependency jsdoc-region-tag to v2 (#134)
- feat: support regapic LRO (#135)
- chore(main): release 2.1.0 (#136)
- chore(deps): update dependency linkinator to v4 (#138)
- fix: better support for fallback mode (#139)
- fix: change import long to require (#140)
- fix: do not import the whole google-gax from proto JS (#1553) (#143)
- fix: allow passing gax instance to client constructor (#144)
- chore(main): release 2.1.1 (#141)
- fix: preserve default values in x-goog-request-params header (#145)
- test: use fully qualified request type name in tests (#148)
- fix(deps): use google-gax v3.5.2 (#152)
- chore(deps): update dependency jsdoc to v4 (#154)
- fix: regenerated protos JS and TS definitions (#155)
- chore(deps): update dependency @types/node to v18 (#150)
- chore(main): release 2.1.2 (#146)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
